clion/src/test/java/org/sonarlint/intellij/clion/BuildWrapperJsonGeneratorTest.java

```diff
@@ -26,6 +26,8 @@
 import java.util.Map;
 import java.util.TreeMap;
 import org.junit.jupiter.api.Test;
+import org.sonarlint.intellij.clion.AnalyzerConfiguration;
+import org.sonarlint.intellij.clion.BuildWrapperJsonGenerator;
```

Comment:
Those added imports look suspicious, are they needed?

Reply:
Unused imports from leftovers after the copy paste, to be removed indeed